### DISCUSSION TITLE
Changes to my github page haven't deployed in 2 days

### DISCUSSION THREAD

**Discussion by aciupan** [community/community#40100] [Projects and Issues]
### Select Topic Area

General

### Body

Hi,

I made slight changes to my github page 2 days ago, and they still haven't deployed. Info here: https://github.com/aciupan/aciupan.github.io/actions

Thoughts?

Thank you!

**Comment by airtower-luna** [comment]
The jobs staying queued for 2 days is definitely not normal. You could try pushing again to kick off another run, that might help if the bug has been fixed meanwhile. If it doesn't help, [send a bug report to GitHub support](https://support.github.com/contact/bug-report) so their engineers can take a look where us users can't.

**Comment by Kirtishukla2004** [comment]
https://maximorlov.com/deploying-to-github-pages-dont-forget-to-fix-your-links/ 
check this link or 
Switch your current Github Pages branch to some other branch and click on Save Switch back to your desirable Github Pages branch and click on Save. This will force Github to update your changes.

**Comment by Kirtishukla2004** [comment]
if these will not work check for incidents on github page i found the people who were going with the same queued error has solved
